compiler/control/OMROptions.cpp

```diff
@@ -732,6 +733,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"enableSpecializedEpilogues",         "O\tapply restore non-volatiles selectively",         SET_OPTION_BIT(TR_EnableSpecializedEpilogues), "F"},
    {"enableTailCallOpt",                  "R\tenable tall call optimization in peephole", SET_OPTION_BIT(TR_EnableTailCallOpt), "F"},
    {"enableThisLiveRangeExtension",       "R\tenable this live range extesion to the end of the method", SET_OPTION_BIT(TR_EnableThisLiveRangeExtension), "F"},
+   {"enableTrampolines",                  "O\tenable trampolines to always be used (zLinux64 and zOS64)", SET_OPTION_BIT(TR_EnableTrampolines), "F"},
```

Comment:
The comment implies this _forces_ trampolines to be used. Is this the case? Reading the code below it seems to imply that trampolines _can_ be used. Perhaps a better comment may be

```
enable generation of trampolines for method dispatch
```

Also while this option is currently used on Linux on z and z/OS there should be nothing preventing us from using the same option on other platforms. Hence this option should be platform agnostic.